docs/src/languages/ansible.md

```diff
@@ -0,0 +1,75 @@
+# Ansible
+
+Support for Ansible in Zed is provided via a community-maintained [Ansible extension](https://github.com/kartikvashistha/zed-ansible).
+
+- Tree-sitter: [zed-industries/tree-sitter-yaml](https://github.com/zed-industries/tree-sitter-yaml)
+- Language Server: [ansible/vscode-ansible](https://github.com/ansible/vscode-ansible/tree/main/packages/ansible-language-server)
+
+## Setup
+
+### File detection
+
+By default, to avoid mishandling non-Ansible YAML files, the Ansible Language is not associated with any file extensions by default.  To change this behavior you can add a `"file_types"` section to the Zed settings inside your project (`.zed/setttings.json`) or your Zed user settings (`~/.config/zed/settings.json`) to match your folder/naming conventions.  For example:
+
+```json
+"file_types": {
+    "Ansible": [
+      "**.ansible.yml",
+      "**.ansible.yaml",
+      "**/defaults/*.yml",
+      "**/defaults/*.yaml",
+      "**/meta/*.yml",
+      "**/meta/*.yaml",
+      "**/tasks/*.yml",
+      "**/tasks/*.yml",
+      "**/tasks/*.yaml",
+      "**/handlers/*.yml",
+      "**/handlers/*.yaml",
+      "**/group_vars/*.yml",
+      "**/group_vars/*.yaml",
+      "**/playbooks/*.yaml",
+      "**/playbooks/*.yml",
+      "**playbook*.yaml",
+      "**playbook*.yml"
+    ]
+  }
+```
+
+Feel free to modify this list as per your needs.
+
+### LSP Configuration
+
+LSP options for this extension can be configured under Zed's settings file. To get the best experience, add the following configuration under the `"lsp"` section in your `~/.zed/settings.json`:
+
+```json
+"lsp": {
+  // Notice how the `ansible` key under `ansible-language-server/settings` is omitted.
+  // This is because it's added from within the extension code, to allow for a cleaner config under Zed's settings file.
```

Comment:
It looks like there is an `ansible` key under `ansible-language-server.settings` so I don't understand this comment.

Reply:
Hey @notpeter - Sure, let me try to explain this better:

If we were configuring the [path option](https://github.com/ansible/vscode-ansible/blob/5a89836d66d470fb9d20e7ea8aa2af96f12f61fb/docs/als/settings.md#L7) of the lsp, then the default configuration key would be: `ansible.ansible.path`. Instead, under `settings`, the users can simply configure `ansible.path` to achieve the same. 

Similarly:
1. `ansible.ansible.useFullyQualifiedCollectionNames` becomes `ansible.useFullyQualifiedCollectionNames` ;
2. `ansible.python.interpreterPath` becomes `python.interpreterPath` 

... and so on. 

Since all ansible options start with the `ansible` key, the extension as such [adds that](https://github.com/kartikvashistha/zed-ansible/blob/06a958ce59a6a58994c3f461b92d1e24846693ba/src/ansible.rs#L101) on your behalf, which imo allows for a much cleaner and manageable config. 

But this is just my opinion - so far, I haven't had anyone raise any issue with this and if upstream (i.e., you folks) wish that I don't do this injection, I can prepare a quick release to reflect that.

Reply:
Excellent. I totally understand and have updated the text in a way that I think makes that clearer.  Feel free to submit follow on PRs if I got it wrong.